src/geophires_x/Economics.py

```diff
@@ -1813,7 +1848,27 @@ def __init__(self, model: Model):
         )
         self.jobs_created = self.OutputParameterDict[self.jobs_created.Name] = OutputParameter(
             Name="Estimated Jobs Created",
-            UnitType=Units.NONE,
+            UnitType=Units.JOBS,
+            CurrentUnits=JobsUnit.JOBS,
+            PreferredUnits=JobsUnit.JOBS
```

Comment:
I don't see the value in defining a specific unit for this - counts are unitless and represented as `Units.NONE` i.e. https://github.com/NREL/GEOPHIRES-X/blob/28df988d6f004c400239383e1f48e2b4bed9adb8/src/geophires_x/SUTRAWellBores.py#L57

Reply:
Yah, I was struggling with the code during the class, so I added this just to get it run. I will set it back to what is right - but this code does work... just doesn't make alot of sense.